### ISSUE TITLE
The browser extension doesn't build by default

### ISSUE THREAD

**Issue by aubincleme** [passbolt/passbolt_browser_extension#66]
## The browser extension doesn't build by default
* Passbolt Version: 2d78f1cf49e1e72963f56680c34ea593202e05c8 / v2.8.3
* Platform and Target:
-- Platform: Linux 4.19.0-2-amd64 #1 SMP Debian 4.19.16-1 (2019-01-17) x86_64 GNU/Linux
-- Node v10.15.3
-- Npm v6.4.1

### What you did
Steps to reproduce : 
 ```
git clone https://github.com/passbolt/passbolt_browser_extension.git
cd passbolt_browser_extension
npm install
./node_modules/.bin/grunt
```
At this point, we see a first build error : 
```
caubin@t580:~/tmp/passbolt_browser_extension (master*=) % ./node_modules/.bin/grunt
Running "copy:background_page" (copy) task
Copied 1 file

Running "copy:content_scripts" (copy) task
Created 10 directories, copied 12 files

Running "browserify:app" (browserify) task
>> Error: Cannot find module '../model/gpgauth' from '/home/caubin/tmp/passbolt_browser_extension/src/all/background_page/event'
Warning: Error running grunt-browserify. Use --force to continue.

Aborted due to warnings.
```

Even though we know that the build won't succeed, we can still try with `grunt --force` : 
```
caubin@t580:~/tmp/passbolt_browser_extension (master*=) % ./node_modules/.bin/grunt --force
Running "copy:background_page" (copy) task
Copied 1 file

Running "copy:content_scripts" (copy) task
Created 10 directories, copied 12 files

Running "browserify:app" (browserify) task
>> Error: Cannot find module '../error/invalidMasterPasswordError' from '/home/caubin/tmp/passbolt_browser_extension/src/all/background_page/event'
Warning: Error running grunt-browserify. Used --force, continuing.
>> Error: Cannot find module '../model/gpgauth' from '/home/caubin/tmp/passbolt_browser_extension/src/all/background_page/event'
Warning: Error running grunt-browserify. Used --force, continuing.
Fatal error: Callback was already called.
```

### What happened
It looks like two files in he repository do not have the correct case:
* `src/all/background_page/error/InvalidMasterPasswordError.js` is using CamelCase instead of lowerCamelCase (the file name should be `invalidMasterPasswordError.js`)
* `src/all/background_page/model/gpgAuth.js` is using lowerCamelCase but shouldn't (the file name should be `gpgauth.js`)

Fixing those two files by either renaming them or creating symlinks also fixes the build.

### What you expected to happen
The build should succeed when cloning the repository.

**Comment by fthzkrtn** [comment]
Worked. Thanks.

**Comment by stripthis** [comment]
Fixed with v2.10.2
